### PR TITLE
Added more unit tests

### PR DESCRIPTION
**Pull request by cyian-1756**
# Category

This change is exactly one of the following (please change `[ ]` to `[x]`) to indicate which:
* [ ] a bug fix (Fix #...)
* [ ] a new Ripper
* [X] a refactoring
* [ ] a style change/fix

# Description

I added unit tests for hentai2read, hentaicafe, myhentaicomics, e-hentai, shesfreaky and sinnercomics

# Testing

Required verification:
* [x] I've verified that there are no regressions in `mvn test` (there are no new failures or errors).
* [x] I've verified that this change works as intended.
  * [x] Downloads all relevant content.
  * [x] Downloads content from multiple pages (as necessary or appropriate).
  * [x] Saves content at reasonable file names (e.g. page titles or content IDs) to help easily browse downloaded content.
* [x] I've verified that this change did not break existing functionality (especially in the Ripper I modified).

Optional but recommended:
* [ ] I've added a unit test to cover my change.

### CODE REVIEW

**Comment by cyian-1756** [comment]
All my changes are passing, the only failed test was a timeout on imgur (Which seems to be having some connection issues right now)

**Comment by metaprime** [comment]
See https://github.com/RipMeApp/ripme/pull/240 -- I've split `BasicRippersTest` into a separate test file per ripper. I think it will make it easier to track test coverage that way.

File: src/test/java/com/rarchives/ripme/tst/ripper/rippers/BasicRippersTest.java [hunk 1]
```diff
@@ -293,9 +330,8 @@ public void testVkPhotosRip() throws IOException {
         testRipper(ripper);
     }
 
-    // https://github.com/RipMeApp/ripme/issues/206 : Disabled test : BasicRippersTest#testXhamsterAlbums -- fix and re-enable
-    // public void testXhamsterAlbums() throws IOException {
-    //     XhamsterRipper ripper = new XhamsterRipper(new URL("https://xhamster.com/photos/gallery/sexy-preggo-girls-9026608"));
-    //     testRipper(ripper);
-    // }
+     public void testXhamsterAlbums() throws IOException {
```
[the remark this answers is not in the document]
Comment (metaprime) [reply src/test/java/com/rarchives/ripme/tst/ripper/rippers/BasicRippersTest.java]: Note: I've enabled this test (in it's new location) here: https://github.com/RipMeApp/ripme/pull/256

**Comment by cyian-1756** [comment]
@metaprime I split my changes into their own files and made a new PR #264

**Pull request closed without merging**